### ISSUE TITLE
TinyGo can't find wasm-ld

### ISSUE THREAD

**Issue by johanbrandhorst** [tinygo-org/tinygo#305]
Running latest TinyGo from `dev`, with all dependencies installed, I get the following error compiling some wasm:

```
tinygo build -o ./html/test.wasm -target wasm --no-debug ./js/main.go
error: failed to link /tmp/tinygo332217407/main: exec: "wasm-ld-8": executable file not found in $PATH
make: *** [Makefile:10: tinygo] Error 1
```

After symlinking `/usr/bin/wasm-ld` to `/usr/bin/wasm-ld-8` it works. I think we should support both paths.

**Comment by johanbrandhorst** [comment]
This seems to be intentionally targetting `wasm-ld-8` only on linux. Is this the name used on Ubuntu and Debian?

https://github.com/tinygo-org/tinygo/blob/fa5df4f524c3b4f15360c2da964932692e3ab4af/commands.go#L9

**Comment by johanbrandhorst** [comment]
We should presumably switch this over at some point? Can we remove this translation?

**Comment by aykevl** [comment]
Well, Debian only has `wasm-ld-8` and not `wasm-ld`. I would propose to add a fallback mechanism, for example by listing multiple alternatives:

```go
"wasm-ld": []string{"wasm-ld-8", "wasm-ld"},
```

I have a partial patch for that lying around here somewhere...

**Comment by aykevl** [comment]
@johanbrandhorst can you try #323?

**Comment by johanbrandhorst** [comment]
Yep, works for me!

**Comment by aykevl** [comment]
This has been fixed in #323.

**Comment by talentlessguy** [comment]
For some reason it keeps happening :/

```
➜ tinygo build -o build/out.wasm -target wasm ./src
error: failed to link /tmp/tinygo705502756/main: none of these commands were found in your $PATH: wasm-ld-10 wasm-ld

➜ tinygo version
tinygo version 0.13.1 linux/amd64 (using go version go1.14.2 and LLVM version 10.0.0)
```

is it worth opening an issue?
